src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Slider.scss';
+
+const Slider: React.FC<React.InputHTMLAttributes<HTMLInputElement>> = (
+	props
+) => {
+	return (
+		<>
+			<div
+				className={`h-[34px] w-full flex items-center justify-center rounded`}
```

Comment:
@manishdoley23 use h-8 here, h-[] format must be avoided.
And only one child is present, remove flex and related classes.